content/en/guides/core/registry/model_registry_eol.md

```diff
@@ -87,6 +136,22 @@ For more details and to learn about the full range of features in the W&B Regist
 
 ## FAQs
 
+### How much notice will we receive before our organization is scheduled for migration?
+
+W&B is committed to providing sufficient notice. Expect to receive direct communication and see an advisory banner in the W&B UI approximately 4-6 weeks before your organization's scheduled migration window. This will give your team time to prepare and ask any questions. 
+
+### How long will copying data from the legacy Model Registry to the W&B Registry take?
+This depends on the volume of data in your legacy Model Registry. During this brief copy period for your specific team, write access to that registry (both the old Model Registry and the new W&B Registry) will be temporarily paused to ensure data integrity. W&B anticipates this specific write-pause to be around an hour or so based on current testing. W&B will provide more specific estimates during targeted communications. Other W&B services will remain fully operational.
+
+### What if we use features like Protected Aliases or Team Service Accounts heavily?
+The migration is designed to handle these. Protected Aliases will have an equivalent in the W&B Registry, and your existing ones will be mapped. Team Service Account permissions will be preserved within the context of the migrated registries.
+
+### How long will we be able to access our legacy Model Registry (MR) after our organization is migrated?
+After your organization's data is successfully copied to the new W&B Registry (GR), the legacy MR will immediately become read-only. You will be able to view your data in the legacy MR in this read-only state for approximately 4 weeks. After this period, the UI views for the legacy MR will be hidden.
```

Comment:
@trane293 We're already hiding the UI for the legacy model registry post migration. Did we want to mention that users can view their data from the `model-registry` project view? Or should we avoid that?